.github/workflows/build.yml

```diff
@@ -32,4 +59,17 @@ jobs:
             ${{ runner.os }}-nuget-
     
       - name: Building AMT
-        run: .\build.ps1 build -Configuration release -Version ${{ env.BUILD_VERSION }} -BuildCounter ${{ github.run_number }}
\ No newline at end of file
+        run: |
+          .\build.ps1 build -Configuration ${{ env.CONFIGURATION }} -Version ${{ env.BUILD_VERSION }} -BuildCounter ${{ github.run_number }}
+        shell: pwsh
```

Comment:
Is this actually working in PowerShell Core? That would be sweet. I don't think the other teams have started testing with PSHell Core yet.

Reply:
Yes, it is working, I'm using Ubuntu. If we don't include the last line it will failed. I ran the build and tests.